### PR TITLE
fix: LEAP-1433: Remove deprecated feature flag FF_DEV_1170

### PR DESCRIPTION
**Pull request by Gondragos**
Eliminated references to the obsolete feature flag `FF_DEV_1170` (`ff_front_1170_outliner_030222_short`) across multiple files, including tests, components, and utilities. Key changes include:
- Updated relevant tests by removing the flag setup and associated logic.
- Refactored components to exclude conditional logic tied to the flag.
- Cleaned up feature-flag utility files by removing the declaration of the flag.

| **Key** | `ff_front_1170_outliner_030222_short` |
|---|---|
| **Variable Name** | `FF_DEV_1170` |
| **Ends up as** | 🟩 `True` |
| **Description** | Outliner + Details |

### CODE REVIEW

**Comment by bmartel** [comment]
@yyassi-heartex and I were just talking about how it would be really amazing to clean this up, awesome 🎉

File: web/libs/editor/tests/e2e/tests/rich-text/rich-text-edge-cases.test.js [hunk 1]
```diff
@@ -2,20 +2,14 @@ const assert = require("assert");
...
-
 const edgeCaseConfig = `<View>
     <Labels name="label" toName="html">
         <Label value="Highlight" background="rgb(255, 64, 182)" />
     </Labels>
     <HyperText name="html" value="$html" />
 </View>`;
 const edgeCaseHtml = `<p>This
-is an <u><span>e</span><span>x</span><span>a</span><span>m</span><span>p</span><span>l</span><span>e</span></u><br/> of 
+is an <u><span>e</span><span>x</span><span>a</span><span>m</span><span>p</span><span>l</span><span>e</span></u><br/> of
```
Comment (Gondragos) [inline web/libs/editor/tests/e2e/tests/rich-text/rich-text-edge-cases.test.js]:
```suggestion
is an <u><span>e</span><span>x</span><span>a</span><span>m</span><span>p</span><span>l</span><span>e</span></u><br/> of 
```

**Review by yyassi-heartex — APPROVED** [review]
excited to see this get cleaned up!

**Comment by borisheartex** [comment]
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14101237319)

File: web/libs/editor/src/components/App/App.jsx [hunk 2]
```diff
@@ -279,56 +275,37 @@ class App extends Component {
...
+                    showComments={store.hasInterface("annotations:comments")}
+                    focusTab={store.commentStore.tooltipMessage ? "comments" : null}
                   >
                     {mainContent}
-                  </SidePanels>
+                    {store.hasInterface("topbar") && <BottomBar store={store} />}
+                  </SideTabsPanels>
                 )
+              ) : isBulkMode ? (
+                <>{mainContent}</>
```
Comment (hlomzik) [inline web/libs/editor/src/components/App/App.jsx]:
```suggestion
                mainContent
```

File: web/libs/frontend-test/src/helpers/LSF/Sidebar.ts [hunk 3]
```diff
@@ -28,15 +25,9 @@ export const Sidebar = {
...
   },
   get regions() {
-    return LabelStudio.getFeatureFlag(FF_DEV_1170).then((isFFDEV1170) => {
-      if (isFFDEV1170) {
-        return this.outliner
-          .should("be.visible")
-          .get(".lsf-tree__node:not(.lsf-tree__node_type_footer) .lsf-tree-node-content-wrapper");
-      }
-
-      return this.legacySidebar.should("be.visible").get(".lsf-region-item");
```
Comment (hlomzik) [inline web/libs/frontend-test/src/helpers/LSF/Sidebar.ts:38]: `legacySidebar` should also be removed then

File: web/libs/editor/tests/e2e/tests/image.selected-region.test.js [hunk 4]
```diff
@@ -44,40 +43,51 @@ async function testRegion(testType, toolAccelerator, I, LabelStudio, AtImageView
...
 
   I.say("There should be no regions initially");
-  AtSidebar.seeRegions(0);
+  AtOutliner.seeRegions(0);
 
   I.say(`${testType} initial region`);
-  await doDrawingAction(I, { msg: `Initial ${testType}`, fromX: 150, fromY: 110, toX: 150 + 50, toY: 110 + 50 });
+  await doDrawingAction(I, {
+    msg: `Initial ${testType}`,
+    fromX: 319 + 150,
```
Comment (hlomzik) [inline web/libs/editor/tests/e2e/tests/image.selected-region.test.js]: would be awesome to see these numbers as consts to understand the meaning behind them

**Review by hlomzik — APPROVED** [review]
That's a titanic job, thank you! Small observations, but minors.

**Comment by Gondragos** [comment]
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14265211737)
> Successfully merged:  create mode 100644 web/libs/ui/src/shadcn.ts

**Comment by Gondragos** [comment]
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14311563556)
> Successfully merged:  create mode 100644 label_studio/users/product_tours/configs/show_ask_ai.yml

**Comment by Gondragos** [comment]
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14333253799)
> Successfully merged:  create mode 100644 docs/themes/v2/source/images/storages/s3-trust-policy.png

File: web/libs/editor/tests/e2e/fragments/AtImageView.js [hunk 5]
```diff
@@ -106,6 +110,7 @@ module.exports = {
 
   setZoom(scale, x, y) {
     I.executeScript(Helpers.setZoom, [scale, x, y]);
+    I.wait(1);
```
Comment (Gondragos) [inline web/libs/editor/tests/e2e/fragments/AtImageView.js:113]:
```suggestion
    I.wait(2);
```
It looks like it may be not enough time to wait

**Comment by Gondragos** [comment]
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/14368999730)
> Successfully merged:  create mode 100644 web/libs/storybook/addons/theme-toggle/theme-toggle.tsx
